package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ibm-ldap",
-  "version": "1.0.1",
+  "version": "2.0.1",
```

Comment:
Move it back to `1.0.1`. I'll bump the version when I merge it using npm's tool for that.